### PR TITLE
New Adapter: Evolution Technologies

### PR DESCRIPTION
**Pull request by imedvedko**
Docs: https://github.com/prebid/prebid.github.io/pull/4421

### CODE REVIEW

**Comment by bsardo** [comment]
Hi @imedvedko. I just wanted to check if there is any relationship between these two because the naming might be a bit confusing to some:
```
	BidderEVolution             BidderName = "e_volution"
	BidderEvolutionTechnologies BidderName = "evolutionTechnologies"
```

**Comment by apykhteyev** [comment]
Hi @bsardo . This company is a customer who's using our solution and wishes to white label it under the name `evolutionTechnologies` because already have some in-house implementation `e_volution`.  Unfortunately, I don't know the current status of their in-house solution, but we were asked to use such a naming. Hope it clarifies

File: static/bidder-info/iionads.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-endpoint: "http://{{.Host}}.iionads.com/openrtb/{{.PublisherID}}"
+endpoint: "http://ads-pbs.iionads.com/openrtb/{{.PublisherID}}?host={{.Host}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/iionads.yaml:1]: This change doesn't seems specific to adding `evolution technologies` adapter

File: static/bidder-info/limelightDigital.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-endpoint: "http://{{.Host}}.ortb.net/openrtb/{{.PublisherID}}"
+endpoint: "http://ads-pbs.ortb.net/openrtb/{{.PublisherID}}?host={{.Host}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/limelightDigital.yaml:1]: similar to https://github.com/prebid/prebid-server/pull/2653/files#r1143367385, change doesn't seems specific to adding evolution technologies adapter

**Review by onkarvhanumante — APPROVED** [review]

**Comment by SyntaxNode** [comment]
> Hi @bsardo . This company is a customer who's using our solution and wishes to white label it under the name `evolutionTechnologies` because already have some in-house implementation `e_volution`. Unfortunately, I don't know the current status of their in-house solution, but we were asked to use such a naming. Hope it clarifies

Could you please reach out to your customer to explain why they need two separate bidders? Per https://github.com/prebid/prebid-server/issues/2654 we're trying to be more consistent across Prebid.org Projects. Do they plan to deprecate `e_volution` in favor of `evolutionTechnologies`?

@bretg This will cause a targeting conflict with the `evolution` bidder in PBS-Java.

**Comment by bretg** [comment]
@imedvedko - please consider a different biddercode, since this name conflicts with an existing code for in the first 6 letters. See https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#choose-a-name

 e.g. How about "evtech" as the bidder code? Shorter is better for ad server targeting.

**Comment by SyntaxNode** [comment]
FWIW: "evtech" sounds great to me.

**Comment by apykhteyev** [comment]
> Hi @bsardo . This company is a customer who's using our solution and wishes to white label it under the name `evolutionTechnologies` because already have some in-house implementation `e_volution`. Unfortunately, I don't know the current status of their in-house solution, but we were asked to use such a naming. Hope it clarifies

Just got info - that the customer wants to keep both adapters for now. I hope there will not be massive issues to keep both.

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**